apps/docs/src/.vitepress/config.ts

```diff
@@ -36,6 +36,11 @@ export default defineConfig({
       { text: "Basics", link: "/basics/", activeMatch: "/basics/" },
       { text: "Tokens", link: "/tokens/introduction", activeMatch: "/tokens/" },
       { text: "Development", link: "/development/", activeMatch: "/development/" },
+      {
+        text: "Resources",
```

Comment:
fyi with thsi new nav bar entry, we get a bug (too many items overall ^^)
https://github.com/orgs/SchwarzIT/projects/5/views/7?layout=board&pane=issue&itemId=51919555
I created this follow up ticket.

Alternatively, to prevent that bug, we might consider putting the resources as an additional category under the development section ¯\_(ツ)_/¯

Reply:
sounds good aswell :)